receive_test.go

```diff
@@ -337,18 +337,19 @@ file zzz.aa
 	assert.Equal(t, ok, false)
 }
 
-func sockPair(ctx context.Context) (Stream, Stream) {
-	c1 := make(chan *types.Packet, 32)
-	c2 := make(chan *types.Packet, 32)
-	return &fakeConn{ctx, c1, c2}, &fakeConn{ctx, c2, c1}
-}
+// func sockPair(ctx context.Context) (Stream, Stream) {
+// 	c1 := make(chan *types.Packet, 32)
+// 	c2 := make(chan *types.Packet, 32)
+// 	return &fakeConn{ctx, c1, c2}, &fakeConn{ctx, c2, c1}
+// }
```

Comment:
Can we remove unused funcs?

Reply:
removed this. left in some comments in benchmark for debug reference